src/main/kotlin/apply/application/ApplicationFormDtos.kt

```diff
@@ -28,9 +22,19 @@ data class UpdateApplicationFormRequest(
 
     val answers: List<AnswerRequest> = emptyList(),
 
-    @field:NotBlank
     val password: String
-)
+) {
+    constructor(
+        recruitmentId: Long,
+        referenceUrl: String,
+        isSubmitted: Boolean,
+        answers: List<AnswerRequest>
+    ) : this(recruitmentId, referenceUrl, isSubmitted, answers, "")
+
+    constructor(
+        recruitmentId: Long
+    ) : this(recruitmentId, "", false, emptyList(), "")
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    constructor(
        recruitmentId: Long,
        referenceUrl: String = "",
        isSubmitted: Boolean = false,
        answers: List<AnswerRequest> = emptyList()
    ) : this(recruitmentId, referenceUrl, isSubmitted, answers, "")
```